SDK Showcase/SDK Showcase/Views/Categories/CategoryView.swift

```diff
@@ -5,47 +5,85 @@ import SwiftUI
     @State private var actions = [Action]()
     @State private var errorValue = ""
     @State private var isProcessing = false
+    @State private var isPresentingSheet = false
     
     var body: some View {
         HStack {
-            List {
-                Text(category.description)
-                    .multilineTextAlignment(.leading)
-                    .font(.system(size: 15))
-                    .foregroundStyle(.secondary)
-                
-                Section(header: Text("Required Actions")) {
-                    ForEach(category.requiredActions, id:\.self) { action in
-                        ActionView(action: binding(for: action))
+            ZStack {
+                List {
+                    Text(category.description)
+                        .multilineTextAlignment(.leading)
+                        .font(.system(size: 15))
+                        .foregroundStyle(.secondary)
+                    
+                    if !category.requiredActions.isEmpty {
+                        Section(header: Text("Required Actions")) {
+                            ForEach(category.requiredActions, id:\.self) { action in
+                                ActionView(action: binding(for: action))
+                            }
+                        }
+                    }
+                    
+                    if !category.optionalActions.isEmpty {
+                        Section(header: Text("Optional Actions")) {
+                            ForEach(category.optionalActions, id:\.self) { action in
+                                ActionView(action: binding(for: action))
+                            }
+                        }
+                    }
+                    
+                    if !category.selection.isEmpty {
+                        Section(header: Text("Select")) {
+                            ForEach(category.selection, id:\.self) { selection in
+                                Button(action: {
+                                    buttonAction(for: selection)
+                                }, label: {
+                                    HStack {
+                                        if let logo = selection.logo {
+                                            Image(systemName: logo)
+                                        }
+                                        Text(selection.name)
+                                    }
+                                })
+                            }
+                            
+                        }
                     }
+                    
+                    Section(content: {
+                        if isProcessing {
+                            ProgressView()
+                                .progressViewStyle(CircularProgressViewStyle())
+                        } else {
+                            TextResult(result: system.isSDKInitialized ? "SDK initialized" : "SDK not initialized \(errorValue)")
```

Comment:
Can we have some constatns? Or `description` defined in `system`?

Reply:
Yes we can, go ahead and create a PR for that

Reply:
Another PR for just introduced code by this PR?

Reply:
If you want to create these consts - yes :) 
If you don't want to, I will make them as costs in the next PR as this is out of this scope (just create a ticket for that)

Reply:
Now it is clear.